ml-agents-envs/setup.py

```diff
@@ -41,7 +41,10 @@ def run(self):
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
     ],
-    packages=["mlagents.envs", "mlagents.envs.communicator_objects"],  # Required
+    packages=[
+        "mlagents_envs",
+        "mlagents_envs.communicator_objects",
+    ],  # Required  # TODO SIDE CHANNELS
```

Comment:
This was a bug waiting to happen - side_channels directory wasn't in the built packages. I'm switching it to automatically find the directories in another PR.